examples/oriented_images.js

```diff
@@ -0,0 +1,107 @@
+/* global itowns, document, renderer, initPicture  */
+// # Simple Globe viewer
+
+// Define initial camera position
+var positionOnGlobe = {
+    longitude: 2.423814,
+    latitude: 48.844882,
+    altitude: 100 };
+
+var promises = [];
+
+// `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+var viewerDiv = document.getElementById('viewerDiv');
+
+// Instanciate iTowns GlobeView*
+var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+    renderer: renderer,
+    handleCollision: false,
```

Comment:
This is not needed, since it's only used when `noControls` isn't set.

Reply:
ok